Tasks/GulpV0/task.json

```diff
@@ -81,9 +81,9 @@
             "name": "gulpjs",
             "type": "string",
             "label": "gulp.js location",
-            "defaultValue": "node_modules/gulp/bin/gulp.js",
-            "required": true,
-            "helpMarkDown": "gulp.js to run when agent can't find global installed gulp.  Defaults to the gulp.js under node_modules folder of the working directory.",
+            "defaultValue": "",
```

Comment:
Is changing the default value considered a breaking change? Would this break customers without gulp globally installed on their agents, who are using the previous default location?

Reply:
It is, now that you mention it. I should at least bump the minor version, not the patch version.

Reply:
Not sure what kind of versioning scheme is used, but if it's semantic versioning, the major would need to be bumped for a breaking change, right? https://semver.org/ Ignore me if this isn't employed though.